backend/src/common/interfaces/repository.interface.ts

```diff
@@ -1,5 +1,5 @@
 type Repository<T = unknown> = {
-    find(): Promise<T>;
+    find(id: number): Promise<T>;
```

Comment:
Name this method findById
```suggestion
    findById(id: number): Promise<T>;
```